docs/faq/FAQ.md

```diff
@@ -0,0 +1,19 @@
+---
+id: FAQ
+title: FAQ
+description: All FAQ related to Questions
+---
+
+## FAQS
```

Comment:
tbh I don't think we need this page, the FAQs will already be listed in the sidebar